src/bower.json

```diff
@@ -7,5 +7,10 @@
     "leaflet": "^0.7.3",
     "multiple-select": "~1.1.0",
     "ramda": "0.8.0"
+  },
+  "overrides": {
+    "leaflet": {
+      "ignore": true
```

Comment:
@ddohler this should make leaflet available to tests, but not include it in the vendor build, since it was easier to include them via CDN. The css is broken if it is included via gulp.